homeassistant/components/kostal_plenticore/select.py

```diff
@@ -0,0 +1,137 @@
+﻿"""Select Kostal Plenticore charging/usage mode"""
+from __future__ import annotations
+
+from abc import ABC
+from datetime import timedelta
+import logging
+
+from homeassistant.components.select import SelectEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import DEVICE_DEFAULT_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import (
+    DOMAIN,
+    SELECT_SETTINGS_DATA,
+)
+from .helper import (
+    SelectDataUpdateCoordinator,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities
+):
+    """Add kostal plenticore Sensors."""
```

Comment:
stale

Reply:
Copied from senor.py to keep things consistent. Why was it ok, when maintainer submitted the code for senor.py?

Reply:
It was OK then because it was about sensors, now it's selects or switches.
Sorry for not being more clear about the comment.

Reply:
I still do not get the point. Looked at knx switch.py how they did it and rebuild it. If this is still "stale" I'm out of ideas ... if not, I will adopt switch.py in the same way.

Reply:
KNX just forgot to update their docstring, a PR fixing that would be welcome.
Anyhow, you fixed it in ca61960